examples/plugins/afk.js

```diff
@@ -0,0 +1,30 @@
+function inject (bot, option) {
+  // it is a good idea to make a same space for your plugin
+  bot.afk = {
+    start,
+    stop
+  }
+
+  let afk; let rotation = 0
+
+  async function start () {
+    afk = setInterval(async () => {
+      if (rotation === 0) {
+        await bot.look(0, 0)
+        rotation = 1
```

Comment:
by making rotation 0/1 you've essentially made a boolean, maybe actually use a boolean to be explicit?